hockeysdk/build.gradle

```diff
@@ -52,7 +52,7 @@ publish {
 }
 
 dependencies {
-    compile fileTree(dir: 'libs', include: ['*.jar'])
+    compile "com.android.support:support-annotations:$supportLibVersion"
```

Comment:
@MatkovIvan please don't introduce the dependency. We don't want to have any dependency to support-annotations. If that's an issue, please get in touch in slack!